src/shared/storage-format/index.ts

```diff
@@ -15,7 +15,25 @@ export const emptyDocument: StorageFormat = {
   variant: variantType,
   version: 0,
   dateModified: getCurrentDatetime(),
-  document: { plugin: 'rows' },
+  document: {
+    plugin: 'type-generic-content',
+    state: {
+      content: {
+        plugin: 'rows',
+        state: [
+          {
+            plugin: 'text',
+            state: [
+              {
+                type: 'p',
+                children: [{ text: '' }],
+              },
+            ],
+          },
```

Comment:
just curious: why do you need to provide an empty state now, where you did not need to before?

Reply:
I got an error in the static renderer 'state not defined' with the previous value here.  
I guess this issue might never happen in production because an empty document is always opened first with the editor (where the missing state is added) and then the static renderer. But it seemed like a good idea to change it anyways to be sure an empty document has a valid form from the start.